docs/Query_Syntax.md

```diff
@@ -97,6 +97,14 @@ As of version 0.21, it is possible to add geo radius queries directly into the q
 
 Radius filters can be added into the query just like numeric filters. For example, in a database of businesses, looking for Chinese restaurants near San Francisco (within a 5km radius) would be expressed as: `chinese restaurant @location:[-122.41 37.77 5 km]`.
 
+## Vector Similarity filter in query
+
+It is possible to add vector similaruty queries directly into the query language.
+The basic syntax is `"*=>[ TOP_K {num|$num} @vector $query_vec]"` for running TOP K query on @vector field.
+<!-- It is also possible to run a query on filtered result, the syntax is `"{some sub-query}=>[ TOP_K {num|$num} @vector $query_vec]"`. -->
```

Comment:
It is also possilbe to run a Hybrid Query on a filtered result. 

A Hybrid query allows the user to specify a filter criteria that ALL results in a TOP K query must satisfy. The filter criteria can only include fields with non-vector indexes (e.g. indexes created on scalar values)

The General syntax is "{some filter query}=>[ TOP_K {num|$num} @vector $query_vec]". For example:
- {@published_year:[2020 2021]} - Only entities published between 2020 and 2021
- =>  - Separates scalar and vector query
- [ TOP_K {num|$num} @vector_field $query_vec] - Return TOP num entities where query_vec is similar to the vector stored in @vector_field